docs/updates/2019-04.md

```diff
@@ -0,0 +1,180 @@
+
+### Default web socket
+
+If you are mainly working with one engine instance, using the new default socket connection feature might be of use.
+By clicking on the star, top right in catwalk, the current web socket will be stored in the browser's local storage and
+used in coming catwalk sessions.
+
+![screenshot](../images/catwalk-default-websocket.png)
+
+### Add support for selections in "always one selected value" fields
+
+A field with the [property](https://core.qlik.com/services/qix-engine/apis/qix/definitions/#nxfieldproperties)
+`qOneAndOnlyOne` will enforce only one selected value. The selection model in catwalk has been updated accordingly.
+
+![screenshot](../images/catwalk-field-select-one-value.gif)
+
+### Visual support for loosely coupled tables
+
+We have added support for loosely couple tables, i.e. tables with a circular reference. Normally this is not intended in
+a data model and catwalk will help out by marking the involved fields with a dashed red line.
+
+![screenshot](../images/catwalk-circular-reference.png)
+
+### Export hypercube data and definition
+
+In one of the incoming feature requests there was an expressed need for exporting hypercube data. It sounded like an
+interesting features so we decided to build support for it but also offer a way to export the hypercube definition to
+Clipboard. Both features can be found in the new icons top right in open hypercubes.
+
+![screenshot](../images/catwalk-export-hypercube.png)
+
+### Save open hypercubes
+
+Something which has irritated us for a while is that all hypercubes created in catwalk gets lost when the window is
+refreshed or closed. To prevent this we now store the hypercube definition in the browser's local storage.
+
+### Keep table headers during scrolling
+
+Previously, when you navigate in large applications, the table headers were lost which caused some confusions on what
+table the fields belonged. With the latest changes we will keep the table names on top as long as there are more fields
+in the table. This is especially good if you have a complex application with a lot of association. In these it isn't
+uncommon to have tables with large gaps between the fields, making it easy to miss the bottom ones.
+
+![screenshot](../images/catwalk-scrolling-headers.gif)
+
+### Various improvements
+
+Apart from the larger fixes we have made a couple of smaller improvements:
+
+* add support for table and field comments
+* handle rendering issues when layout isn't retrieved and while loading the layout
+* Multiple table-names properly displayed in tool tips
+* Fix key field sorting during selection in other fields
+
+## corectl 1.0.0 beta release
+
+The rest of the team worked on reaching a 1.0.0 version of corectl. An API specification was created, commands
+scrutinized and inconsistencies fixed throughout the commands. In the final work we decided upon going from verb-noun to
```

Comment:
"In the final work" sounds weird/swenglish. Maybe "Ultimately, we decided to change the command structure from noun-verb to verb-noun to better [...]". I think a few examples here would be nice since it shouldn't really be a requirement to know grammar terminology to use corectl or read these updates :)